engine/table/src/main/java/io/deephaven/engine/table/impl/replay/DataDrivenReplayer.java

```diff
@@ -20,8 +22,33 @@ public DataDrivenReplayer(DateTime startTime, DateTime endTime) {
     }
 
     @Override
-    public DateTime currentTime() {
-        return currentTime;
+    public long currentTimeMillis() {
```

Comment:
Maybe we can move this up a level to avoid repetition, delegating to an abstract accessor for `currentTime`.

Reply:
So, the hierarchy right now is a bit confusing, with DataDrivenReplayer and FixedStepReplayer (extends io.deephaven.engine.table.impl.replay.Replayer) having completely separate overrides than the parent class.

I _think_ it could be cleaned up, but it would be a pretty major gutting that would be better done in isolation IMO.